Description: Fix interpreter for perl scripts
 Policy 10.4 states that Perl scripts must use
 /usr/bin/perl directly and not via /usr/bin/env
Author: Valentin Vidic <Valentin.Vidic@CARNet.hr>
Last-Update: 2018-08-17
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/lib/plugins/stonith/external/vcenter
+++ b/lib/plugins/stonith/external/vcenter
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 #
 # External STONITH module for VMWare vCenter/ESX
 #
